son/content/intermediate_outcomes/work_in_early_adulthood_(25_to_29_years)/occupational_level_of_young_people_aged_25_to_29_years/3.0.md

```diff
@@ -93,13 +93,36 @@ Young people with a disability were more likely to be in a lower working class o
                 "code": "IN33",
                 "title": "Percentage of 25 to 29 year olds in higher professional positions, by socio-economic background (UK, 2014 to 2024, 3-year averages)",
                 "tooltipTitle": "",
-                "data": "2.0/IN33-2.0-occupational-level--by-SEB--chart-format.csv",
+                "data": [{
+                    "label": "Total",
+                    "data": "3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--total--chart-format.csv"
+                },
+                {
+                    "label": "Higher professional",
+                    "data": "3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--higher-professional--chart-format.csv"
+                },
+                {
+                    "label": "Lower professional",
+                    "data": "3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--lower-professional--chart-format.csv"
+                },
+                {
+                    "label": "Intermediate",
+                    "data": "3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--intermediate--chart-format.csv"
+                },
+                {
+                    "label": "Higher working",
+                    "data": "3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--higher-working--chart-format.csv"
+                },
+                {
+                    "label": "Lower working",
+                    "data": "3.0/IN33-3.0-occupational-level-of-young-people-aged-25-to-29-years--by-moving-average--lower-working--chart-format.csv"
+                }],
+                "toggle": "radio",
                 "height": 450,
                 "type": "bary",
-                "xkey": "seb",
+                "xkey": "average_window",
                 "ykey": "value",
-                "zkey": "category",
-                "sort": "category",
+                "zkey": "tertiary_split_value",
                 "scale": "%",
                 "rounding": "0dp",
                 "xgrid": false,
```

Comment:
On line 132, we have `"xtitle": "Socio-economic background"`.
I think this should be `"Year"` or `"Time period"`